src/components/navigation/navigation-item/NavigationItem.tsx

```diff
@@ -161,11 +161,13 @@ export class NavigationItem extends Component<NavigationItemProps, NavigationIte
     return <img {...imageProps} />;
   };
 
-  private _renderShowMoreLessButton = () => {
+  private _renderShowMoreLessButton = (title: string) => {
     if (!this.props.data.previewImage) {
       return null;
     }
     const {expandText} = this.state;
+    const readMoreLessLabel = expandText ? this.props.readLessLabel : this.props.readMoreLabel
+    const readMoreLessLabelAria = (readMoreLessLabel).charAt(readMoreLessLabel.length - 1) === 's' ? readMoreLessLabel.substring(0, readMoreLessLabel.length-1) : readMoreLessLabel;
```

Comment:
this will work only for En and other langs where "s" define plural.
Instead use Pluralization from preact-i18n.

Reply:
addressed.
more button-
![image](https://user-images.githubusercontent.com/79077248/235933491-07543d24-e511-40d8-9eeb-e26b8f4d3d35.png)

chapter button in filter bar-
![image](https://user-images.githubusercontent.com/79077248/235933690-f4d9645c-e6e8-425b-b1e9-5ef0ab4e2b01.png)